charts/sequencer/templates/_helpers.tpl

```diff
@@ -71,3 +71,7 @@ name: {{ .Values.moniker }}-sequencer-metrics
 {{/* New sequencer address */}}
 {{- define "sequencer.address"}}{ "bech32m": "{{ . }}" }
 {{- end }}
+
+{{/* uint64 fee converted to a astria proto Uint128 with only lo set */}}
+{{- define "sequencer.fee"}}{ "lo": {{ . }} }
```

Comment:
I don't know that this helper name is accurate. `sequencer.toUint128Proto`?

Reply:
Yeah, we'd ultimately end up doing that. I had a dislike for extra configuration data being passed in "from the outside". I would like all genesis settings relevant to the app/sequencer to be in one place.

We would then either enforce that the cometbft chain ID match the sequencer chain ID - or ignore cometbft.

Reply:
I don't think your comment is on the correct note here?

Reply:
ODd, yeah. This was supposed to go on another comment. I remember explicitly putting it there.. :-/

Reply:
Renamed the helper in [dfdff4d](https://github.com/astriaorg/astria/pull/1346/commits/dfdff4db0976afd50485d2d8b523f86526929987)